src/main/java/creativei/service/impl/InquiryServiceImpl.java

```diff
@@ -111,4 +120,66 @@ public Inquiry updateAll(List<Inquiry> inquiries) {
         Long cityId = filterVo.getCityVo().getId();
         return inquiryCustomDao.findByFilters(status, caseIndex, cityId);
     }
+
+    @Override
+    public ResponseObject getInquiryCount() throws ParseException {
+        InquiryCountVo inquiryCountVo=null;
+        try {
+            inquiryCountVo= new InquiryCountVo();
+            inquiryCountVo.setDailyCount(getDailyCount());
+            inquiryCountVo.setWeekCount(getWeeklyCount());
+            inquiryCountVo.setMonthCount(getMonthlyCount());
+            inquiryCountVo.setHotLeadCount(getHotLeadCount());
+            inquiryCountVo.setEnrolledCount(getEnrolledCount());
+            return ResponseObject.getResponse(inquiryCountVo);
+        }catch (Exception e){
+            logger.error("Cause: "+e.getCause().getCause().getMessage());
+            return ResponseObject.getResponse(ExceptionType.RESULTSET_GENERATION_EXCEPTION.getMessage(),ExceptionType.RESULTSET_GENERATION_EXCEPTION.getCode());
+        }
+    }
+
+
+    private Integer getDailyCount() throws ParseException {
+        logger.info("getDailyCount method");
+        String date1= LocalizationUtil.getFormattedDate(new Date());
+        Date inquiryToDate=LocalizationUtil.stringToDateConverter(date1);
+        Date inquiryFromDate=new Date();
+        return inquiryCustomDao.findCountByInquiryDate(inquiryToDate,inquiryFromDate);
+    }
+
+    private Integer getWeeklyCount() throws ParseException {
+        logger.info("getWeeklyCount method");
+        Date date=new Date();
+        Date inquiryToDate=LocalizationUtil.getWeekStartingDate(date);
+        Date inquiryFromDate=date;
```

Comment:
Use calendar class object as all these are already provided there.

Reply:
Calendar object being used in LocalisationUtil class